src/multi-selector.css

```diff
@@ -2,13 +2,14 @@
 	display: inline-block;
 	position: relative;
 	font-size: 1.4rem;
-}
-
-.cpr-multi-selector {
```

Comment:
there were two `.cpr-multi-selectors` right next to each other so I combined them.